keyboards/handwired/technicpad/info.json

```diff
@@ -0,0 +1,42 @@
+{
+    "manufacturer": "nwhirschfeld",
+    "keyboard_name": "TechnicPad",
+    "maintainer": "nwhirschfeld",
+    "bootloader": "caterina",
+    "bootmagic": {
+        "matrix": [0, 0]
+    },
+    "build": {
+        "lto": true
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["D1"],
+            ["D0"],
+            ["D4"],
+            ["C6"]
+        ]
+    },
+    "processor": "atmega32u4",
```

Comment:
```suggestion
```

Reply:
solved in https://github.com/qmk/qmk_firmware/pull/23238/commits/02352065d1f847a4a689679db243adc3d4d59e00